docs/docusaurus.config.js

```diff
@@ -1,108 +1,163 @@
+	label: 'Legacy 1.x',
+	to: 'https://legacy-docs-v1.rasa.com',
 };
 
+// FIXME: when deploying this for real, change to '/docs/rasa/'
+const BASE_URL = '/docs/rasa/next/';
+const SITE_URL = 'https://rasa.com';
+
 module.exports = {
-  title: 'Rasa Open Source Documentation',
-  tagline: 'The tagline of my site',
-  url: 'https://your-docusaurus-test-site.com',
-  // FIXME: when deploying this for real, change to '/docs/rasa/'
-  baseUrl: '/docs/rasa/next/',
-  favicon: 'img/favicon.ico',
-  organizationName: 'RasaHQ',
-  projectName: 'rasa',
-  themeConfig: {
-    navbar: {
-      title: 'Rasa Open Source',
-      logo: {
-        alt: 'Rasa',
-        src: 'https://rasa.com/static/60e441f8eadef13bea0cc790c8cf188b/rasa-logo.svg',
-      },
-      items: [
-        {
-          label: 'Docs',
-          to: '/', // "fake" link
-          position: 'left',
-          items: versions.length > 0 ? [
-            {
-              label: versions[0],
-              to: '/',
-              activeBaseRegex: versions[0],
-            },
-            ...versions.slice(1).map((version) => ({
-              label: version,
-              to: `${version}/`,
-              activeBaseRegex: version,
-            })),
-            {
-              label: 'Master/Unreleased',
-              to: 'next/',
-              activeBaseRegex: `next`,
-            },
-            legacyVersion,
-          ] : [
-            {
-              label: 'Master/Unreleased',
-              to: '/',
-              activeBaseRegex: `/`,
-            },
-            legacyVersion,
-          ],
-        },
-        {
-          href: 'https://github.com/rasahq/rasa',
-          label: 'GitHub',
-          position: 'right',
-        },
-      ],
-    },
-    footer: {
-      style: 'dark',
-      copyright: `Copyright © ${new Date().getFullYear()} Rasa Technologies GmbH`,
-    },
-    gtm: {
-      containerID: 'GTM-PK448GB',
-    },
-  },
-  themes: [
-    ['@docusaurus/theme-classic', {
-      customCss: require.resolve('./src/css/custom.css'),
-    }],
-    path.resolve(__dirname, './themes/theme-live-codeblock'),
-  ],
-  plugins: [
-    ['@docusaurus/plugin-content-docs', {
-      // https://v2.docusaurus.io/docs/next/docs-introduction/#docs-only-mode
-      routeBasePath: '/',
-      // It is recommended to set document id as docs home page (`docs/` path).
-      homePageId: 'index',
-      sidebarPath: require.resolve('./sidebars.js'),
-      editUrl: 'https://github.com/rasahq/rasa/edit/master/docs/',
-      remarkPlugins: [
-        [ remarkCollapse, { test: '' }],
-        remarkSources,
-        remarkProgramOutput
-      ],
-    }],
-    ['@docusaurus/plugin-sitemap', {
-      cacheTime: 600 * 1000, // 600 sec - cache purge period
-      changefreq: 'weekly',
-      priority: 0.5,
-    }],
-    path.resolve(__dirname, './plugins/google-tagmanager'),
-  ],
+	title: 'Rasa Open Source Documentation',
+	tagline: 'The tagline of my site',
```

Comment:
I've merged a fix for this in master

Reply:
OK I'll revert this

Reply:
I've changed this to match master, where these two properties have the same value. I don't think that's the intention of the tagline 🤔  though